dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/enrollment/EnrollmentServiceTest.java

```diff
@@ -793,6 +792,55 @@ void shouldFailWhenRequestingListOfEnrollmentsAndAtLeastOneNotAccessible() {
         exception.getMessage());
   }
 
+  @Test
+  void testGetEnrollmentById() {
+    manager.save(enrollmentA);
+    manager.save(enrollmentB);
+    assertEquals(enrollmentA, manager.get(Enrollment.class, enrollmentA.getUid()));
+    assertEquals(enrollmentB, manager.get(Enrollment.class, enrollmentB.getUid()));
+  }
+
+  @Test
+  void testGetEnrollmentsByTrackedEntityProgramAndEnrollmentStatus()
+      throws ForbiddenException, BadRequestException {
+    manager.save(enrollmentA);
+    Enrollment enrollment1 = createEnrollment(programA, trackedEntityA, orgUnitA);
```

Comment:
Mixing enrollments with numeric and alphabetical suffixes confuses me. What is the reason? How do they relate to each other?